organization/philanthropy.md

```diff
@@ -0,0 +1,34 @@
+---
+type: resource
+explains: our policy on philanthropy
+---
+
+# Philanthropy policy
+
+The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html). All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations) (ANBI).
+
+We will be philanthropically funded until we have enough members paying dues to support a sustainable blended financial model.
+
+[See our financial model](https://about.publiccode.net/organization/financial-model.html) for more information about our long term financial plans.
+
+## Working together with philanthropists
+
+We value the experience, expertise and connections of the philanthropists who've chosen to support us. Our donors have helped us scale, grow internationally and communicate for impact.
+
+Philanthropists are not involved in our prioritization of activities or day to day operations.
+
+[See our governance model](https://about.publiccode.net/organization/governance-model.html) for more information about how the Foundation is governed.
+
+## Donors
+
+To date, we've received donations from:
```

Comment:
Great! Do we also want to disclose amounts? Or do that in the financial reporting when that comes?

Reply:
We thought the annual report, and possibly press releases, was a great place for this.